### PR TITLE
Cleanup ocp db

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
Looks good

File: lib/public/DB.php [hunk 1]
```diff
@@ -1,64 +0,0 @@
...
-	 * @param string $query Query string
-	 * @param int $limit Limit of the SQL statement
-	 * @param int $offset Offset of the SQL statement
-	 * @return \OC_DB_StatementWrapper prepared SQL query
-	 *
-	 * SQL query via Doctrine prepare(), needs to be execute()'d!
-	 * @deprecated 8.1.0 use prepare() of \OCP\IDBConnection - \OC::$server->getDatabaseConnection()
-	 * @since 4.5.0
-	 */
-	static public function prepare( $query, $limit=null, $offset=null ) {
```
Comment (MorrisJobke) [inline lib/public/DB.php]: *cough*

bildschirmfoto 2018-03-26 um 14 58 17

*cough*

Comment (nickvergessen) [reply lib/public/DB.php]: Will fix the activity tests...

Comment (rullzer) [reply lib/public/DB.php]: Done

**Pull request merged**